### ISSUE TITLE
UNUSED_LOCAL_VARIABLE false positive with var used in cfoutput inside cffunction

### ISSUE THREAD

**Issue by zspitzer** [cflint/CFLint#464]
```
<cfcomponent>
<cffunction name="test" access="public" returnType="any" output="true">
	<cfscript>
		var resultText="";
	</cfscript>
	<cfset resultText="error">
	<cfsetting showdebugoutput="false" />
	<cfcontent type="text" reset="yes"><cfoutput>#resultText#</cfoutput><cfabort>
</cffunction>
</cfcomponent>```

**Local variable resultText is not used in function test, consider removing it.**

**Comment by ryaneberly** [comment]
fixed in dev branch

**Comment by Vintic** [comment]
This doesn't work, when var is used in HASH:
`<cfcomponent displayname="myComponent">
    <cffunction name="getData">
        <cfset var testVar = 123>

        <cfsavecontent variable = "sql_query">
            <cfoutput>
                SELECT #testVar#
            </cfoutput>
        </cfsavecontent>
    </cffunction>
</cfcomponent>
`
